x11-misc/xkblayout-state/xkblayout-state-20171214.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+# Commit Date: 27 Nov 2017
+COMMIT="d023bd7f58a54da08c3aad3c234a629b8a010892"
+
+DESCRIPTION="A small program to get/set the current XKB layout"
+HOMEPAGE="https://github.com/nonpop/xkblayout-state"
+SRC_URI="${HOMEPAGE}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+RDEPEND="x11-libs/libX11"
+DEPEND="${RDEPEND}"
+
+src_compile() {
+	emake CXX=$(tc-getCXX)
```

Comment:
Quote it, tc-getCXX can return string with spaces.

Reply:
Done.